creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
 
 # Differences Between Minecraft: Bedrock Edition and Minecraft: Java Edition
 
-On the surface, Bedrock Edition and Java Edition seem very similar, but under the hood is a completely different story. The different code bases create distinct development environments. This tutorial outlines the major differences you as a content creator should be aware of.
+On the surface, Bedrock Edition and Java Edition may seem very similar, but under the hood is a completely different story. The different code bases create distinct development environments for content creators. This tutorial outlines the major differences you as a content creator should be aware of.
 
 In this tutorial you will learn the following:
 
 > [!div class="checklist"]
 >
-> - A brief history of Java Edition and Bedrock edition.
+> - A brief history of Java Edition and Bedrock Edition.
 > - How the two editions differ and what it means for content creation.
 
 There are two major versions of Minecraft.
 
 ### Minecraft: Java Edition
 
-This version was originally released in 2009. This version used to be called Minecraft until it was renamed to Minecraft: Java Edition in September 2017. As the name implies, it’s developed in Java and isn’t compatible with the current version of Minecraft for the most part. This edition is commonly referred to as simply ***Java***.
+This is the original version of Minecraft, first released in May 2009. In September 2017, coinciding with the launch of Bedrock Edition, the "Java Edition" subtitle was added to the title. As the name implies, this version of Minecraft is developed in the [Java programming language](https://en.wikipedia.org/wiki/Java_(programming_language)). It's available only on Windows, Mac, and Linux PCs. This edition is commonly referred to as simply ***Java***.
 
 ### Minecraft: Bedrock Edition
 
-Bedrock Edition was launched on Sept. 20, 2017 and was based on Minecraft: Pocket Edition, which was released in 2011. It brought together nine of the major device platforms under a singular codebase called the Bedrock Engine. This was a rewrite of Minecraft from the ground up and brought along with it some fundamental changes to the platform paving the way for an exciting new development community. This edition is commonly referred to as simply ***Bedrock***.
+Bedrock Edition was fully launched in September, 2017, based on Minecraft: Pocket Edition, which was released in 2011. It brought together nine of the major device platforms under a singular codebase called the Bedrock Engine. This was a rewrite of Minecraft from the ground up using the [C++ programming language](https://en.wikipedia.org/wiki/C%2B%2B). It's available on Windows 10 PCs, Xbox, PlayStation, Nintendo Switch, Android, and iOS. This edition is commonly referred to as simply ***Bedrock***.
 
```

Comment:
Included platform and language information just to complement the likewise information in the above paragraph. Also "exciting new development community" ok yeah I know this is the bedrock docs website but let's try to at least pretend to be unbiased here